terraform/central-account/README.md

```diff
@@ -1,16 +1,10 @@
 # Demo infrastructure for ConsoleMe in AWS
 
-Since Consoleme is not public yet, we have to create a tar.gz file in the root directory, have Terraform upload the tarball to an S3 bucket, and then download it as part of the demo infrastructure.
+- Download and install Terraform 0.15.0. I highly recommend [tfenv](https://github.com/tfutils/tfenv)
 
-- First, create the tarball of Consoleme in the root directory of this repository.
-
-```bash
-make consoleme.tar.gz
-```
-
-Note that if you modify this consoleme.tar.gz file and then run Terraform again, it will update the Terraform infrastructure to include your changes.
-
-- Download and install Terraform 0.13.4. I highly recommend [tfenv](https://github.com/tfutils/tfenv)
+  - If using tfenv, run the following commands in the `central_account` directory:
+    - `tfenv install 0.15.0`
```

Comment:
Would this doc get old as new minor tf versions arrive? We might want to use [latest:\<regex\>](https://github.com/tfutils/tfenv#tfenv-use-version) syntax here

Reply:
Addressed this, please re-check!